updatehub/Cargo.toml

```diff
@@ -20,7 +20,7 @@ chrono = { version = "0.4", default-features = false, features = ["serde"] }
 cloud = { path = "../updatehub-cloud-sdk", package = "updatehub-cloud-sdk" }
 compress-tools = "0.3"
 derive_more = { version = "0.99", default-features = false, features = ["deref", "deref_mut"] }
-easy_process = "0.1"
+easy_process = "0.2"
```

Comment:
This commit must come before the use of the new function.

Reply:
It's only newer on the commit's creation date. In the git's history it's right before 139be12a4a77c3b4e05099ecedb10f276fb30601